scripts/linters/pylint_extensions.py

```diff
@@ -1959,6 +2026,386 @@ def process_tokens(self, tokens):
+                single_line_import = False
+            if inside_multi_line_import_scope and token == ')':
+                inside_multi_line_import_scope = False
+
+        return (
+            single_line_import, import_line_num, inside_multi_line_import_scope
+        )
+
+    def check_comment_is_present_with_object_class(self, tokens):
+        """Checks whether the object class in a module has been documented
+        or not. If the object class is not documented then adds an error
+        message.
+
+        Args:
+            tokens: Token. Object to access all tokens of a module.
+        """
+        object_comment_present = False
+        outside_function_signature_block = True
+        outside_args_section = True
+        object_present_inside_arg_section = False
+        object_present_inside_return_section = False
+        object_present_in_function_signature = False
+        object_already_encountered_line_num = 0
+        args_section_end_line_num = 0
+        object_comment_line_num = 0
+
+        object_type_regex = r'# Here we use object because'
+
+        for (token_type, token, (line_num, _), _, line) in tokens:
+            line = line.strip()
+
+            if token_type == tokenize.COMMENT:
+                if re.search(object_type_regex, line):
+                    object_comment_present = True
+                    object_comment_line_num = line_num
+
+            # Checking if linters are in argument-section, return-section or
+            # outside of the function definition.
+            # Eg:
+            #       <outside function signature block>
+            #   def func(<argument-section>) -> <return-section>:
+            #       <outside function signature block>.
+            if token_type == tokenize.NAME:
+                if token == 'def':
+                    outside_function_signature_block = False
+                    func_def_start_line = line_num
+                    outside_args_section = False
+
+            if token_type == tokenize.OP:
+                if token == '->':
+                    outside_args_section = True
+                    args_section_end_line_num = line_num
+                if outside_args_section and token == ':':
+                    outside_function_signature_block = True
+
+            # Checking if object type is present in function signature or not.
+            if token_type == tokenize.NAME and token == 'object':
+                if not outside_args_section:
+                    object_present_inside_arg_section = True
+                elif (
+                    outside_args_section and
+                    args_section_end_line_num == line_num
+                ):
+                    object_present_inside_return_section = True
+
+            if (
+                object_present_inside_arg_section or
+                object_present_inside_return_section
+            ):
+                object_present_in_function_signature = True
```

Comment:
We also have this code for `Any`, again, please separate this into helper function.

Reply:
Done, Now `Any` and `object` both calling the same helper function.